### DISCUSSION TITLE
Use rectangle to span 2 of 3 monitors on macbook pro laptop

### DISCUSSION THREAD

**Discussion by ljpanetta** [rxhanson/RectanglePro-Community#352] [General]
Hi there,

I have a Macbook Pro laptop with the Apple M2 silicon chip with two external monitors connected via the two Thunderbolt ports. For work purposes, I have to use Microsoft Remote Desktop Program (RDP) to log into my server. I have my laptop in the middle with an external monitor to the left and another external monitor to the right of the laptop.

I want the RDP to span across the two external monitors, but I want to use my laptop screen for local programs like iMessage, News app, etc. So, I want RDP to span across 2 of the 3 monitors.

I am able to do this on my Windows laptop by editing the RDP configuration file. However, I can't get it to work on my Macbook. I purchased Rectangle Pro hoping that I can utilize this to span across the two external monitors, but I don't know how to do it in Rectangle Pro.

Any suggestions? Thank you in advance for the consideration.

Larry

**Comment by rxhanson** [comment]
Thanks for purchasing. Currently there is not a way to do this in Rectangle or Rectangle Pro. I have a backlog item specifically for this, but don't have a timeframe on when I can get to it.
